### DISCUSSION TITLE
The Return of the Comment(s)

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#9] [Comments]
# The Return of the Comment(s)

Father, husband, developer relations and web standards expert, and cat demo builder.

http://localhost:8888/2024/01/02/the-return-of-the-comments/

**Comment by cfjedimaster** [comment]
Hi, this is Raymond, testing things out. :)

**Comment by robertz** [comment]
Very nice

**Comment by JonasEriksson** [comment]
Yay! Maybe you'll now no longer doubt as much whether people read your blog posts - because we do! :-) All the best for 2024!

**Comment by cfjedimaster** [comment]
Nice - it appears as if the widget here auto-updates. I didn't have to reload to see the new comments. (I think so anyway.) And I'm now noticing my comments are labeled "Owner", which is nice.

**Reply by recursivecodes** [reply]
😻

**Comment by ramibch** [comment]
Nice!

```python

def say_hello():
    print("hello")
```

I can comment in Markdown as well 🙂

**Reply by cfjedimaster** [reply]
+1 for the Python. ;)

**Comment by orcpac7** [comment]
Been following you since (our) early ColdFusion days and though we’ve taken different paths, I still enjoy your writings and glimpses of different technologies and techniques!
# Happy New Year!

**Reply by orcpac7** [reply]
AKA Jeff Davis

**Comment by cfjedimaster** [comment]
FYI for folks curious, I'm also getting emails for every comment, which is perfect. And I guess expected, but I was still surprised to see it.

**Comment by mpoint** [comment]
Thanks for sharing.

**Comment by hryggrbyr** [comment]
Well, this is cool! 😃

**Comment by bobmonsour** [comment]
Nicely done, Raymond! Looks good.

**Comment by ecarlisle** [comment]
Very cool! Will have to check out the JS load options. Remembering "D" and many hacks to defer loading.

**Comment by AlHolden** [comment]
Wait a sec... so the excellent and free open source project requires that I tip off the not-so-open-source Redmond project to my presence here? 
I'm not sure if I should be Dennis Hopper paranoid (lot) or Eric Cartman paranoid (not).

**Comment by kegg** [comment]
That's awesome! I was wondering where the comment system had gone. :-) Now I know. Very cool! Love the blog keep it up!

**Comment by Akbarsait** [comment]
Recently I switched my from blogCFC to 11ty and started using utterances for comments. Both Utterances and giscus are amazing.

**Comment by Sepremex** [comment]
Wait a mint, you guys have blogs and comments!, I don't even have time to blog.
